### PR TITLE
MAINT: Display return values as percentages in perf attrib tables

### PR DESCRIPTION
**Pull request by yankees714**
Before:
screen shot 2017-11-10 at 12 15 01 pm

After:
screen shot 2017-11-10 at 12 15 26 pm

### CODE REVIEW

File: pyfolio/utils.py [hunk 1]
```diff
@@ -192,6 +192,11 @@ def print_table(table, name=None, fmt=None, header_rows=None):
         Formatter to use for displaying table elements.
         E.g. '{0:.2f}%' for displaying 100 as '100.00%'.
         Restores original setting after displaying.
+    formatters : list or dict, optional
```
Comment (vikram-narayan) [inline pyfolio/utils.py]: should we combine this and `fmt` above, or rename one of them? it's slightly confusing to have them both

**Comment by jzhang18** [comment]
this looks great!

**Comment by yankees714** [comment]
Update screenshot:
screen shot 2017-11-10 at 3 19 37 pm

File: pyfolio/utils.py (hunk 1 above)
Comment (yankees714) [reply pyfolio/utils.py]: From what I can tell, all the existing call sites that pass `fmt` are passing `'{0:.2f}'` to format float values. `pd.DataFrame.to_html()` accepts a `float_format` arg, so what would you think of  renaming `fmt` to `float_format`, and just forwarding that to `to_html()`? The call sites would then pass `'{0:.2f}.format'`, and the behavior should be the same.

Comment (vikram-narayan) [reply pyfolio/utils.py]: 👍

**Comment by twiecki** [comment]
Nice!!

**Pull request merged**